Formula/tfel.rb

```diff
@@ -0,0 +1,60 @@
+class Tfel < Formula
+  desc "Code generation tool dedicated to material knowledge for numerical mechanics"
+  homepage "https://thelfer.github.io/tfel/web/index.html"
+  url "https://github.com/thelfer/tfel/archive/refs/tags/TFEL-4.0.tar.gz"
+  sha256 "ed3fb2f59c6b8c9896606ef92276f81942433dd5f60d8130ba07c3af80b039e2"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/tfel.git", using: :git, branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.9" => :build
+  fails_with gcc: "5"
+
+  def install
+    args = ["-DUSE_EXTERNAL_COMPILER_FLAGS=ON",
+            "-Denable-reference-doc=OFF",
+            "-Denable-website=OFF",
+            "-Dlocal-castem-header=ON",
+            "-Denable-fortran=OFF",
+            "-Denable-python=ON",
+            "-Denable-python-bindings=OFF",
+            "-Denable-cyrano=ON",
+            "-Denable-lsdyna=ON",
+            "-Denable-aster=ON",
+            "-Denable-abaqus=ON",
+            "-Denable-calculix=ON",
+            "-Denable-comsol=ON",
+            "-Denable-diana-fea=ON",
+            "-Denable-ansys=ON",
+            "-Denable-europlexus=ON"]
```

Comment:
This leads to less noisy `git diff`s.
```suggestion
    args = [
      "-DUSE_EXTERNAL_COMPILER_FLAGS=ON",
      "-Denable-reference-doc=OFF",
      "-Denable-website=OFF",
      "-Dlocal-castem-header=ON",
      "-Denable-fortran=OFF",
      "-Denable-python=ON",
      "-Denable-python-bindings=OFF",
      "-Denable-cyrano=ON",
      "-Denable-lsdyna=ON",
      "-Denable-aster=ON",
      "-Denable-abaqus=ON",
      "-Denable-calculix=ON",
      "-Denable-comsol=ON",
      "-Denable-diana-fea=ON",
      "-Denable-ansys=ON",
      "-Denable-europlexus=ON",
    ]
```